### PR TITLE
Add support for markdown-it-multimd-table

### PR DESCRIPTION
**Pull request by hammurapi**
This pullrequest adds the markdown-it-multimd-table to the project. (See Issue #42)

### CODE REVIEW

File: src/plugin/plugins.ts [hunk 1]
```diff
@@ -26,4 +27,5 @@ export var plugins: markdowItPlugin[] = [
     { plugin: require('markdown-it-mark') },
     { plugin: require('markdown-it-deflist') },
     { plugin: MarkdownItExportHelper }
+    { plugin: MarkdownItMultiMdTable }
```
Comment (qjebbs) [inline src/plugin/plugins.ts]:
```suggestion
    { plugin: require('markdown-it-multimd-table'), params: [{ enableMultilineRows: true }] },
```
You don't have to create a plaugin file `markdownItMultiMdTable.ts`.

**Comment by qjebbs** [comment]
The current table formate feature doesn't compatible to this new table syntax, I'll take a look later.

File: src/plugin/plugins.ts (hunk 1 above)
Comment (hammurapi) [reply src/plugin/plugins.ts]: You are right the single line solution is much nicer than the extra plugin file

**Comment by qjebbs** [comment]
You may need to remove `markdownItMultiMdTable.ts`, it's not used. And I need to update table formatting feature before I can release.

File: src/plugin/markdownItMultiMdTable.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { MarkdownIt } from 'markdown-it';
+import * as container from 'markdown-it-multimd-table';
+
+export function MarkdownItMultiMdTable(md: MarkdownIt) {
+    md.use(container, { enableMultilineRows: true });
+}
```
Comment (qjebbs) [inline src/plugin/markdownItMultiMdTable.ts]: This file is not used, please remove it.

**Comment by qjebbs** [comment]
Sorry, I can't wait anymore, since it only one line of code. But still, thank you.

**Pull request closed without merging**